ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -413,6 +413,12 @@ void HardwareInterface::read(const ros::Time& time, const ros::Duration& period)
     readData(data_pkg, "tool_temperature", tool_temperature_);
     readData(data_pkg, "robot_mode", robot_mode_);
     readData(data_pkg, "safety_mode", safety_mode_);
+    readData(data_pkg, "output_double_register_0", output_double_register_[0]);
+    readData(data_pkg, "output_double_register_1", output_double_register_[1]);
+    readData(data_pkg, "output_double_register_2", output_double_register_[2]);
+    readData(data_pkg, "output_double_register_3", output_double_register_[3]);
+    readData(data_pkg, "output_double_register_4", output_double_register_[4]);
+    readData(data_pkg, "output_double_register_5", output_double_register_[5]);
```

Comment:
See https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/160#issuecomment-615733315 for how you should adapt this to something which *may* work.